src/main/config/config.c

```diff
@@ -723,44 +725,54 @@ void activateConfig(void)
 #endif
 }
 
-void validateAndFixConfig(void)
+static void featureValidateSet(uint32_t mask)
```

Comment:
I wanted the name to reflect that it is only to be used for the validation, it is also static and for local use only. Is it okay to change function names if they are used in other modules? That is the main reason i kept the original name `featureSet()`